src/pages/Admin/AdminMainLayout.tsx

```diff
@@ -13,11 +14,18 @@ const Container = styled.div`
   background: ${({ theme }) => theme.background.secondary}; ;
 `;
 
+const Section = styled.section`
+  display: flex;
+  flex-direction: column;
+  flex-basis: 1 auto;
+  width: 80vw;
```

Comment:
wasnt this the one that's causing the breadcrumb not flexible when the screen width is changed?

Reply:
The section causing my issues the other day was in the BreadcrumbMenu.styled file. The snippet with the max-width: 100% is below.

`export const BreadcrumbArea = styled.div`
  background: ${({ theme }) => theme.background.primary};
  margin: 1.5em;
  height: 90px;
  border-radius: 1em;
  max-width: 100%;`
  
 I will update here, as well, however!

Reply:
All set, thanks for the help in our call.